src/Adapter/Filesystem/SatelliteBuilder.php

```diff
@@ -100,16 +100,15 @@ public function build(): Satellite\SatelliteInterface
             $composer,
         );
 
+        foreach ($this->files as $file) {
+            $satellite->withFile($file);
+        }
+
         if ($this->composerJsonFile !== null) {
             $satellite->withFile($this->composerJsonFile);
             if ($this->composerLockFile !== null) {
                 $satellite->withFile($this->composerLockFile);
             }
-
-            // FIXME: finish the Sylius API client migration
-            $composer->addGithubRepository('sylius-api-php-client', 'git@github.com:gplanchat/sylius-api-php-client.git');
-
-            $composer->install();
```

Comment:
from what I see this case is only for when the user has set from_local to true. So a composer install should never be executed, because the user should do it himself.